hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/Scheduler.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.tpc;
+
+import com.hazelcast.internal.tpc.iobuffer.IOBuffer;
+
+public interface Scheduler {
```

Comment:
Need some JavaDocs, it's quite important component (yeah I know it's just moved, just marking that - as part of our internal API - it should be documented for easier reasoning in the future)

Reply:
Done